lib/pageflow/ability_mixin.rb

```diff
@@ -35,6 +35,10 @@ def pageflow_default_abilities(user)
 
       can :view, [Admin::MembersTab, Admin::RevisionsTab]
 
+      can :delete_own, ::User do |current_user|
+        Pageflow.config.authorize_user_deletion.call(current_user) == true
+      end
+
```

Comment:
Yes need. With calls of the form `authorized?(:delete_own_user, ::User)` CanCan will ignore the block. See `can?(:read, Account)`